File: LABORATORIOS/LABORATORIO3/Ejercicio7/Java/UnirNumeros.java
import java.util.Scanner;

public class UnirNumeros {
    public static int unirNumeros(int x, int y) {
        String strX = Integer.toString(x);
        String strY = Integer.toString(y);
        String strZ = strX + strY;
        int z = Integer.parseInt(strZ);
        return z;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese el primer número: ");
        int x = scanner.nextInt();
        System.out.print("Ingrese el segundo número: ");
        int y = scanner.nextInt();
        int z = unirNumeros(x, y);
        System.out.println("El tercer número obtenido es: " + z);
    }
}

File: LABORATORIOS/LABORATORIO3/Ejercicio7/Python/UnirNumeros.py
def unir_numeros(x, y):
    str_x = str(x)
    str_y = str(y)
    z = int(str_x + str_y)
    return z

x = int(input("Ingrese el primer número: "))
y = int(input("Ingrese el segundo número: "))
z = unir_numeros(x, y)
print("El tercer número obtenido es:", z)